modules/openxr/SCsub

```diff
@@ -18,10 +19,12 @@ env_openxr.Prepend(
         thirdparty_dir + "/src",
         thirdparty_dir + "/src/common",
         thirdparty_dir + "/src/external/jsoncpp/include",
-        thirdparty_dir + "/src/loader",
     ]
 )
 
+if env["platform"] == "android":
```

Comment:
So what I meant here was moving the whole 
```
if env["platform"] == "android":
    # may need to set OPENXR_ANDROID_VERSION_SUFFIX
    env_thirdparty.AppendUnique(CPPDEFINES=["XR_OS_ANDROID", "XR_USE_PLATFORM_ANDROID"])

    # may need to include java parts of the openxr loader
elif env["platform"] == "linuxbsd":
    env_thirdparty.AppendUnique(CPPDEFINES=["XR_OS_LINUX"])

    if env["x11"]:
        env_thirdparty.AppendUnique(CPPDEFINES=["XR_USE_PLATFORM_XLIB"])

    # FIXME: Review what needs to be set for Android and macOS.
    env_thirdparty.AppendUnique(CPPDEFINES=["HAVE_SECURE_GETENV"])
elif env["platform"] == "windows":
    env_thirdparty.AppendUnique(CPPDEFINES=["XR_OS_WINDOWS", "NOMINMAX", "XR_USE_PLATFORM_WIN32"])
```
Block above `env_thirdparty = env_openxr.Clone()` and applying it to `env_openxr`.

This will mean that the correct defines are also in use when we include our openxr headers into our module, I suspect running into this eventually for WMR support. 

It also prevents adding the same defines twice.

Reply:
Updated as requested. Could you please check it?

Reply:
Yup, perfect!